nss-day-cohort-48/fun-website-1-the-single-coders
# Updated HTML for Photos

I updated the index.html file for added photos

##Steps to Test
1. Pull this branch
2. Open in Chrome
3. Verify everything looks right.
